core/chain-configs/src/test_utils.rs

```diff
@@ -253,7 +253,8 @@ pub struct TestClientConfigParams {
     pub min_block_prod_time: u64,
     pub max_block_prod_time: u64,
     pub num_block_producer_seats: NumSeats,
-    pub archive: bool,
+    pub use_split_store: bool,
+    pub is_cloud_archival_writer: bool,
```

Comment:
The naming is inconsistent and confused me a bit. How about `is_split_store` and `is_cloud_archival_writer`? Or some other prefix like `configure_`, `setup_`, `set_` or similar? Anything works as long as it's consistent :)